examples/glue_benchmark/glue.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import csv
+import os
+
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from absl import app
+from absl import flags
+from tensorflow import keras
+
+import keras_nlp
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    "task_name",
+    "mrpc",
+    "The name of the GLUE task to finetune on.",
+)
+
+flags.DEFINE_integer(
+    "batch_size",
+    32,
+    "Batch size of data.",
+)
+
+flags.DEFINE_integer(
+    "epochs",
+    2,
+    "Number of epochs to run finetuning.",
+)
+
+flags.DEFINE_float(
+    "learning_rate",
+    5e-5,
+    "Learning rate",
+)
+
+flags.DEFINE_string(
+    "submission_directory",
+    None,
+    "The directory to save the glue submission file.",
+)
+
+
+flags.DEFINE_string(
+    "finetuning_model_save_path",
```

Comment:
`saved_model_directory`?

If provided, the final fine-tuned model will be saved in the specified directory.